### PR TITLE
feat: Resolver support

### PR DESCRIPTION
**Pull request by mdupont**
Add support for Resolver operations, the last piece that was missing to support all the v1.0 API elements.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbp/nexus-sdk",
-  "version": "1.0.0-beta.20",
+  "version": "1.0.0",
```
Comment (julienmachon) [inline package.json]: I don't think we should change packages versions in PRs like this. We should create a PR with a package bump after the tag/release process. 

Also, I don't think we've reached 1.0.0 yet

Comment (mdupont) [reply package.json]: Resolvers were the missing part before we cover v1.0 API. We already have views, schemas, realms...

Also, the next step would be to tackle AggregateSparqlViews which is a v1.1 feature, so we need to release v1.0.0 before

Comment (julienmachon) [reply package.json]: We don't support cwrite operations on views and we don't support any permission based operation yet...

Comment (mdupont) [reply package.json]: So we are likely to never have a v1.0.0 release and skip right to the v1.1.0 one

**Comment by julienmachon** [comment]
We need to update the README as well to show that resolvers are supported and put some examples with the methods signatures

File: package.json (hunk 1 above)
Comment (julienmachon) [reply package.json]: I guess we need to put more effort in SDK!

**Pull request merged**